Dockerfile

```diff
@@ -1,26 +1,35 @@
-FROM quay.io/ebi-ait/ingest-base-images:python_3.7-alpine
+FROM quay.io/ebi-ait/ingest-base-images:python_3.7-slim
 LABEL maintainer="hca-ingest-dev@ebi.ac.uk"
 
-RUN apk update && \
-    apk add build-base && \
-    apk add openssl-dev && \
-    apk add libffi-dev && \
-    apk add git
+RUN apt-get update 
+#    apk add build-base && \
+#    apk add openssl-dev && \
+#    apk add libffi-dev && \
+#    apk add git
 
 RUN mkdir /app
 WORKDIR /app
 
 COPY requirements.txt ./
-RUN pip install -r requirements.txt
 
-COPY config.py data_archiver.py listener.py ./
+COPY main.py ./
+COPY data ./data
 
+ENV INGEST_API=http://localhost:8080
```

Comment:
ENV setup should be at the top (or high up) of the Docker file.
Code copying should be lower, so we take advantage of docker caching and have faster builds.

Reply:
Moved pip upgrade at the top as well.